go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/aquasecurity/table v1.8.0
 	github.com/aquasecurity/testdocker v0.0.0-20240730042311-4642e94c7fc8
 	github.com/aquasecurity/tml v0.6.1
-	github.com/aquasecurity/trivy-checks v1.3.0
+	github.com/aquasecurity/trivy-checks v1.3.2-0.20241129051814-d0839ca85a45
```

Comment:
Bump up to latest release once https://github.com/aquasecurity/trivy-checks/pull/295 is merged and new trivy-checks is released.